tests/tests.ts

```diff
@@ -124,7 +150,7 @@ describe('multicastStream', function() {
 		const nStreams = 3;
 		for (let i = 0; i < nStreams; ++i) {
 			toStreams.push(
-				new MockWritable({ highWaterMark: hwm / (i + 1) }, i * 10),
+				new MockWritable({ highWaterMark: Math.ceil(hwm / (i + 1)) }, i * 10),
```

Comment:
This change prevents a non-integer `highWaterMark` value like 16384 / 3 = 5461.3. Oddly the test was passing under MacOS and Linux, but failing under Windows.